src/System.Runtime.InteropServices/src/System/Runtime/InteropServices/RuntimeEnvironment.cs

```diff
@@ -23,7 +23,10 @@ public static bool FromGlobalAccessCache(System.Reflection.Assembly a)
         }
         public static string GetRuntimeDirectory()
         {
-            return Path.GetDirectoryName(typeof(object).Assembly.Location) + Path.DirectorySeparatorChar;
```

Comment:
@AtsushiKan 
-> This is, assuming, that this is even what we want GetRuntimeDirectory() to return in an AOT world...
It's not clear what GetRuntimeDirectory should return , my understanding is that it should return the location of coreclr.dll / mrt100 and the current implementation  is not correct if that's the case. Now in AOT world mr100 can be in system32 directory.